geopaparazzispatialitelibrary/src/eu/geopaparazzi/spatialite/database/spatial/util/SpatialiteUtilities.java

```diff
@@ -43,12 +43,74 @@
      * Array of fields that will be ingored in attributes handling.
      */
     public static String[] IGNORED_FIELDS = {SPATIALTABLE_ID_FIELD, "PK_UID", "_id"};
-
+     
     /**
      * Name/path separator for spatialite table names.
      */
     public static final String UNIQUENAME_SEPARATOR = "#"; //$NON-NLS-1$
 
+    /**
+     * Active Connection to a  Spatialite Database
+    * <p/>
+     * <p>Indented for general use for Transformation etc.
+     */
+    private static Database dbSpatialite=null;
+    /**
+     * Active Connection to a  Spatialite Database
+    * <p/>
+     * <p>Indented for general use for Transformation etc.
+     */
+    private static Database dbMemSpatialite=null;
```

Comment:
Do we really want to support memory databases? I can't find a good use for that on android. Also because the OS might kill your app while you are logging due to memory issues. 
Should be better removed?